x/dao/migrations/v3/params.go

```diff
@@ -0,0 +1,30 @@
+package v3
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/planetmint/planetmint-go/x/dao/types"
+)
+
+func MigrateParams(ctx sdk.Context, storeKey storetypes.StoreKey, cdc codec.BinaryCodec) error {
+	store := ctx.KVStore(storeKey)
+	paramBytes := store.Get(types.KeyPrefix(types.ParamsKey))
+
+	var params types.Params
+	err := cdc.Unmarshal(paramBytes, &params)
+	if err != nil {
+		return err
+	}
+
+	params.ValidatorPopReward = 100000000
```

Comment:
the global value from https://github.com/planetmint/planetmint-go/blob/197ebe1279746d79e1f38f8b8ef0c05f813a91e2/util/rddl_token.go#L8
 could be used instead of writing down the complete value